build.gradle

```diff
@@ -248,8 +240,7 @@ test {
     minHeapSize = "1G"
     maxHeapSize = "4G"
 
-
-    if (CI == "true" && CLOUD == "false" ) {
+    if (TEST_VERBOSITY == "minimal") {
```

Comment:
What I meant was that it's not clear to me how `CI == "true" && CLOUD == "false"` in the previous version maps to `TEST_VERBOSITY == "minimal"` in the current version, so I was just curious.